framework/test/unit/modules/interoperability/base_cross_chain_update_command.spec.ts

```diff
@@ -451,7 +445,8 @@ describe('BaseCrossChainUpdateCommand', () => {
 			expect(command['_tokenMethod'].initializeUserAccount).not.toHaveBeenCalled();
 		});
 
-		it('should terminate the chain and add an event when ccm format is invalid', async () => {
+		// decoding
```

Comment:
I think we can remove this comment, either we can encapsulate tests within describe with topic or `it('...')` description are good enough?